.devcontainer/Dockerfile

```
@@ -52,13 +53,26 @@ RUN <<-```
 ENV PATH="/opt/google-cloud-sdk/bin:$PATH"
 
 SHELL ["/bin/bash", "-c"]
-WORKDIR /root/.mcp/superset
-RUN <<-```
+WORKDIR /root/.mcp
+RUN <<-EOF
     set -ex
-    git clone --depth 1 https://github.com/openops-cloud/superset-mcp .
-    wget -qO- https://astral.sh/uv/install.sh | sh && source $HOME/.local/bin/env
+    wget -qO- https://astral.sh/uv/install.sh | sh
+    export PATH="$HOME/.local/bin:$PATH"
+
+    # Superset MCP
+    git clone --depth 1 https://github.com/openops-cloud/superset-mcp superset
```

Remove this. It can cause problems if the commit we are checking out is not the most recent one.

```suggestion
    git clone https://github.com/openops-cloud/superset-mcp superset
```